src/main/scala/com/gu/fastly/Lambda.scala

```diff
@@ -56,19 +66,51 @@ class Lambda {
     dotcomSurrogateKey
   }
 
-  private def sendFastlyPurgeRequestAndAmpPingRequest(contentId: String, purgeType: PurgeType, serviceId: String, surrogateKey: String, fastlyApiKey: String): Boolean = {
-    if (sendFastlyPurgeRequest(contentId, purgeType, serviceId, surrogateKey, fastlyApiKey))
-      sendAmpPingRequest(contentId)
-    else
-      false
+  // Left[Id] for Delete, Right[Id] for Update
```

Comment:
maybe it would be clearer to have a Delete/Update sealed trait thingy/enum rather than using an either?

Reply:
Good idea